spec/models/course/publishable_spec.rb

```diff
@@ -32,7 +32,7 @@
     end
 
     it 'is not publishable' do
-      course.accrediting_provider.not_an_accredited_provider!
+      course.accrediting_provider.toggle(:accredited)
```

Comment:
I think it's better to be explicit here. `toggle` is a stateful method. We should be explicit in setting its value.

```suggestion
      course.accrediting_provider.accredited = false
```